packages/EasyErrorHandler/src/Response/ErrorResponseFactory.php

```diff
@@ -13,13 +13,15 @@
 
 final class ErrorResponseFactory implements ErrorResponseFactoryInterface, FormatAwareInterface
 {
+    private const SUPPORTED_FORMAT = 'json';
```

Comment:
Don't like :-)

Reply:
Why? In this way we can see the actual format by the const name. Otherwise I should go to the `SUPPORTED_FORMAT` declaration to see what is the format under it.

Reply:
Done.